<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="contextMenuPolicy">
       <enum>Qt::DefaultContextMenu</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">QTabWidget::tab-bar {alignment: center;}</string>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="processTab">
       <attribute name="title">
        <string>Процессы</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLineEdit" name="searchLineEdit">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="ProcQTableWidget" name="processTableWidget">
          <property name="contextMenuPolicy">
           <enum>Qt::CustomContextMenu</enum>
          </property>
          <property name="styleSheet">
           <string notr="true">QTableWidget::item:selected { background-color: #FF4800; color: white} QHeaderView::section {background-color: #FEFEFE; padding: 4px; border: 1px solid #F8F8F8;}</string>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <property name="verticalScrollMode">
           <enum>QAbstractItemView::ScrollPerPixel</enum>
          </property>
          <property name="horizontalScrollMode">
           <enum>QAbstractItemView::ScrollPerPixel</enum>
          </property>
          <property name="showGrid">
           <bool>false</bool>
          </property>
          <attribute name="horizontalHeaderHighlightSections">
           <bool>false</bool>
          </attribute>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <column>
           <property name="text">
            <string>Name</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>PID</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>CPU</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>PPID</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Virtual memory</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Resident memory</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>CMD</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>State</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Start time</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Priority</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="resoursesTab">
       <attribute name="title">
        <string>Ресурсы</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="LoadCustomPlot" name="cpuGraph" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="FSTab">
       <attribute name="title">
        <string>Файловые системы</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="FSQTableWidget" name="fileSystemTableWidget">
          <property name="styleSheet">
           <string notr="true">QTableWidget::item:selected { background-color: #FF4800; color: white} QHeaderView::section {background-color: #FEFEFE; padding: 4px; border: 1px solid #F8F8F8;}</string>
          </property>
          <property name="verticalScrollBarPolicy">
           <enum>Qt::ScrollBarAsNeeded</enum>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="defaultDropAction">
           <enum>Qt::CopyAction</enum>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <property name="verticalScrollMode">
           <enum>QAbstractItemView::ScrollPerPixel</enum>
          </property>
          <property name="horizontalScrollMode">
           <enum>QAbstractItemView::ScrollPerPixel</enum>
          </property>
          <property name="showGrid">
           <bool>false</bool>
          </property>
          <attribute name="horizontalHeaderCascadingSectionResizes">
           <bool>false</bool>
          </attribute>
          <attribute name="horizontalHeaderHighlightSections">
           <bool>false</bool>
          </attribute>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <column>
           <property name="text">
            <string>Name</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Mount</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Type</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Total</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Free</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Available</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Used</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ProcQTableWidget</class>
   <extends>QTableWidget</extends>
   <header>procqtablewidget.h</header>
  </customwidget>
  <customwidget>
   <class>FSQTableWidget</class>
   <extends>QTableWidget</extends>
   <header>fsqtablewidget.h</header>
  </customwidget>
  <customwidget>
   <class>LoadCustomPlot</class>
   <extends>QWidget</extends>
   <header>loadcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
